src/components/nuopc/data_comps/dice/cime_config/config_archive.xml

```diff
@@ -0,0 +1,15 @@
+<components version="2.0">
+  <comp_archive_spec compname="dice" compclass="ice">
+    <rest_file_extension>r</rest_file_extension>
```

Comment:
My testing indicated that you also needed the rs1 listed as a rest_file_extension.